qubes/firewall.py

```diff
@@ -422,9 +503,9 @@ def load_v1(self, xml_root):
         policy_v1 = xml_root.get('policy')
         assert policy_v1 in ('allow', 'deny')
         if policy_v1 == 'allow':
-            self.policy = Action('accept')
+            policy = Action('accept')
         else:
-            self.policy = Action('drop')
+            policy = Action('drop')
```

Comment:
I disagree, because the reader wonders, what for this variable really is. Could `default_policy_is_accept` (with `bool()` values) be a replacement?